dataland-frontend/src/assets/content.json

```diff
@@ -0,0 +1,193 @@
+{
+    "pages": [
+        {
+            "title": "Landing Page",
+            "url": "/",
+            "sections": [
+                {
+                    "title": "Welcome to Dataland",
+                    "text": [
+                        "This is the main introduction of our homepage."
+                    ],
+                    "image": [
+                        "/static/logos/gfx_logo_dataland_orange_S.svg"
+                    ]
+                },
+                {
+                    "title": "Intro",
+                    "text": [
+                        "Liberate Data - ",
+                        "Empower Autonomy.  ",
+                        "The alternative to data monopolies."
+                    ],
+                    "image": [
+                        "/static/logos/Dataland_D_svg_Logo.svg"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "/static/icons/Search.svg",
+                            "text": "Search for a company to view or request data..."
+                        }
+                    ]
+                },
+                {
+                    "title": "Quotes",
+                    "text": [
+                        "START YOUR DATALAND JOURNEY"
+                    ],
+                    "cards": [
+                        {
+                            "icon": "GaUte84IN0g",
+                            "title": "XXX XXX,",
```

Comment:
This should contain the name of the CEO from Metzler, right?   Are we still waiting for this info?

Reply:
I just got it yesterday and I will take care of it today